content/04.results.md

```diff
@@ -15,35 +15,43 @@ Panel B shows the types and amounts of articles that we have used for analyses.
-The author metadata of research-related articles was analyzed over the same period as the news-related articles and totaled 13,414 articles.
-Since the demographic makeup of authors published in _Nature_ may not represent overall science authorship, we also analyzed 36,000 randomly selected _Springer_-published articles from English language journals over the same time.
+All identified DOI's were queried using the _Springer_ API to attain all authors' names, positions, and affiliations, however last authors were used as the primary comparator.
+
+To determine if the quoted speakers, mentioned countries, and cited authors in the news-related articles have a similar demographic makeup as the scientists who publish their primary research in _Nature_, we used the all authors' names, positions, and affiliations of Research Articles and Letters published by _Nature_ over the same time period (Figure {@fig:fig1}a, dark orange box).
+Again, last authors were used as the primary comparator.
+The author metadata of research-related _Nature_ articles from 2005 to 2020 totaled 13,414 articles.
+To more broadly represent the overall science authorship, we also separately analyzed 36,000 randomly selected _Springer_-published articles from English language journals over the same time.
+It should be noted that extracted quotes may come from multiple types of persons, such as academic scientists, clinicians, the broader scientific community, politicians, and more.
+However, through anecdotal observation we believe that most sources come from either academic scientists or those actively involved in science.
 The extracted author affiliations from both data sources were mapped to a country using OpenStreetMap.
 Similarly, author names were uniformly processed and then used to predict both gender and name origin.
 
-When considering all _Nature_ articles, news and research, across all time, we find significant variability in the number of articles by type (Figure {@fig:fig1}b).
-The top three observed article frequencies are Research (which includes Letters and Articles), News, and News Feature.
-Since _Nature_ merged Letters and Research articles in 2019, we combine them in our analysis.
-The changing classification of article types may also explain temporal changes in news articles.
-Over time, the frequency of news-related articles is decreasing; however, more specific article types are increasing.
-We observe this in the introduction of the new category "Career Feature" in 2017.
-These articles were previously identified as "Features" articles before 2017 (https://www.nature.com/nature/articles?type=feature).
+The top three observed article frequencies are "Research" (including "Letters" and "Articles"), "News", and "News-Feature".
+Since _Nature_ merged "Letters" and "Research" articles in 2019, we combined them in our analysis.
+We observed significant variability in the number of _Nature_ news-articles by type between 2005 and 2020 (Figure {@fig:fig1}b).
```

Comment:
```suggestion
We observed substantial variability in the number of _Nature_ news-articles by type between 2005 and 2020 (Figure {@fig:fig1}b).
```

Reply:
i think nature's style guide reserves the word "significance" for something with a statistical test - iirc